app/script/conversation/ConversationRepository.js

```diff
@@ -1267,14 +1267,32 @@ z.conversation.ConversationRepository = class ConversationRepository {
    * @returns {Promise} Resolves when conversation was renamed
    */
   renameConversation(conversation_et, name) {
-    return this.conversation_service.updateConversationProperties(conversation_et.id, {name}).then(response => {
+    return this.conversation_service.updateConversationName(conversation_et.id, name).then(response => {
       if (response) {
         amplify.publish(z.event.WebApp.EVENT.INJECT, response, z.event.EventRepository.SOURCE.BACKEND_RESPONSE);
         return response;
       }
     });
   }
 
+  /**
+   * Set the global message timer
+   *
+   * @param {Conversation} conversationEntity - Conversation to rename
```

Comment:
`Conversation to rename` does not match in the case of `updateConversationMessageTimer`